.env.pantheon

```diff
@@ -0,0 +1,13 @@
+DB_NAME=$_ENV['DB_NAME']
+DB_USER=$_ENV['DB_USER']
+DB_PASSWORD=$_ENV['DB_PASSWORD']
+
+# Pantheon sets these values for you. If you want to shuffle them you can do so via your dashboard.
+AUTH_KEY=$_ENV['AUTH_KEY']
+SECURE_AUTH_KEY=$_ENV['SECURE_AUTH_KEY']
+LOGGED_IN_KEY=$_ENV['LOGGED_IN_KEY']
+NONCE_KEY=$_ENV['NONCE_KEY']
+AUTH_SALT=$_ENV['AUTH_SALT']
+SECURE_AUTH_SALT=$_ENV['SECURE_AUTH_SALT']
+LOGGED_IN_SALT=$_ENV['LOGGED_IN_SALT']
+NONCE_SALT=$_ENV['NONCE_SALT']
```

Comment:
Would we want to put WP_HOME and WP_SITEURL in here, too, instead of in `application.php`? 
this mostly applies if we are advising people to edit `application.php` which, as I say elsewhere, I think we should maybe not do and instead instruct them to edit `wp-config.php`. My thought here is that Bedrock's addition of the `/wp` directory breaks a lot of stuff, and if a user saw it and said "that's not right" or "I don't want to log in at /wp/wp-admin, I'll just change this" they could break their site.

Reply:
I did [try this](https://github.com/pantheon-systems/wordpress-composer-managed/pull/9/commits/1b7c26fe39d54319d0ac963b42dcbf9a534c1bf2), but was getting a fatal syntax error (don't remember what is was off the top of my head) when trying to load the site. Maybe you'll notice what I did wrong 🤞.

Also team edit `wp-config`, and am working on some comments to indicate that to users.